toptal/picasso
# feat(Input): add highlightAutofill prop

[FX-2887]

### Description

Add `highlightAutofill` property to `Input`. It is expected to be used only internally by `@toptal/picasso-forms`, that is why I left out any comment to not have it in storybook docs.

### How to test

- check [storybook](https://picasso.toptal.net/fx-2887-highlight-preselected/?path=/story/forms-input--input)
- check the naming of new prop

### Screenshots

![image](https://user-images.githubusercontent.com/6830426/224256547-cfdfdbc5-38f7-400d-bad0-7d7889d74c31.png)

### Development checks

  - left out on purpose

**Breaking change**

-  codemod is created and showcased in the changeset
-  test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-2887]: https://toptal-core.atlassian.net/browse/FX-2887?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
